src/jekyll/_plugins/wf/WFFeedPage.rb

```diff
@@ -15,7 +15,8 @@
 module Jekyll
 
   class WFFeedPage < LanguagePage
-
+    ## TODO: Handle both atom.liquid and rss.liquid
+    ## TODO: Remove support for feed.liquid
```

Comment:
Quick update - checked the last 10 days of H5R Updates
30280 requests for rss.xml - 70.3% of all feed requests
12755 requests for atom.xml - 29.6% of all feed requests